release_notes/AMLS-5835.txt

```diff
@@ -1 +0,0 @@
- + [AMLS-5835] - 'Updated links that look like buttons to properly mimic the behaviour of a button element'
```

Comment:
This is strange... Where is the release note for this change?

Reply:
Re-added this